src/main/resources/liquibase/crop_changelog/19_4_0.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cvterm" columnName="is_system"/>
+			</not>
+		</preConditions>
+		<comment>Adding is_system column</comment>
+        <sql>
+			ALTER TABLE cvterm ADD COLUMN is_system INT(11) NOT NULL DEFAULT '0' AFTER is_relationshiptype;
+        </sql>
+    </changeSet>
+
+	<changeSet author="cuenyad" id="v19.4.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(1) > 0 FROM cvterm where is_system = 0 and name in ('TRIAL_INSTANCE','LOCATION_ID','LOCATION_NAME','EXPT_DESIGN','FIELDMAP COLUMN'
```

Comment:
I think that this precondition can be simplified, if is_system exists, then the variables can be marked as is_system. I dont think it is necessary to check if the value is 0